packages/hardhat-core/src/internal/hardhat-network/provider/modules/eth.ts

```diff
@@ -851,6 +852,11 @@ export class EthModule {
       }
 
       if (error.message.includes("EIP155")) {
+        console.warn(
```

Comment:
I'd just add it to the normal log flow. This `console.warn` can bring problems, like ignoring the loggingEnabled setting.